### DISCUSSION TITLE
BIG SCREEN Blaseball - Feedback and Suggestions

### DISCUSSION THREAD

**Discussion by hgmich** [holmesmr/Blaseball-Userstyles#10] [Ideas]
If you have any suggestions, improvements or feedback for BIG SCREEN Blaseball, please discuss them here!

Please don't post bug reports here! Please [file an issue](holmesmr/Blaseball-Userstyles/issues/new) if you're having a problem with the layout of BIG SCREEN Blaseball.

**Comment by JesamJD** [comment]
there's a new feature when a game is fully loaded - bases, balls, strikes, outs - called maximum blaseball, that puts an overlay around the match's box, and in BSB the text overlaps
![maximumblaseball non-bsb](https://user-images.githubusercontent.com/79963044/109765001-2947c600-7bec-11eb-8e73-ef44013d2b1e.jpg)
![maximimblaseball bsb](https://user-images.githubusercontent.com/79963044/109765027-2ea51080-7bec-11eb-952c-314b405fb048.png)

maybe make the gap between the teams and the game a bit larger? idk

edit: oh shoot, sorry, managed to miss the "please don't file bug reports here," sorry...

**Comment by hadecake** [comment]
This is a great style! Exactly what I was looking for, thank you. 
A suggestion would be to have the scroll bar fade-out when not in use. I keep the tab in windowed so the TV's overscan doesn't clip the edges and the scroll bar is always showing.  Even better would be to add some padding on the sides so it doesn't risk clipping in the fist place. Also a way to turn off the toast notifications in the bottom right. So you can display this on a big screen without having to clear them after every game.
EDIT:     added ~30 px of padding to the top section and now it displays nicely in fullscreen. Not sure if that's the solution but it works on my machine, haha! 
thanks again

**Comment by nanomachinegun** [comment]
hello! I love this stylesheet and it makes Blaseball so much easier to engage with.

The addition of 4 new teams means the 2 extra games get shuffled onto a third row, but I changed .GameWidget to this and everything is still readable with no overlap as far as I can see. (20% is now 16%)
```
.GameWidget {
        flex: 0 1 16%;
```
